docs/pt/docs/tutorial/response-model.md

```diff
@@ -0,0 +1,482 @@
+
+### Use o parâmetro `response_model_exclude_unset`
+
+Você pode definir o parâmetro *decorador de operação de caminho* `response_model_exclude_unset=True`:
+
+=== "Python 3.6 e superior"
+
+    ```Python hl_lines="24"
+    {!> ../../../docs_src/response_model/tutorial004.py!}
+    ```
+
+=== "Python 3.9 e superior"
+
+    ```Python hl_lines="24"
+    {!> ../../../docs_src/response_model/tutorial004_py39.py!}
+    ```
+
+=== "Python 3.10 e superior"
+
+    ```Python hl_lines="22"
+    {!> ../../../docs_src/response_model/tutorial004_py310.py!}
+    ```
+
+e esses valores padrão não serão incluídos na resposta, apenas os valores realmente definidos.
+
+Portanto, se você enviar uma solicitação para essa *operação de caminho* para o item com ID `foo`, a resposta (sem incluir valores padrão) será:
+
+```JSON
+{
+    "name": "Foo",
+    "price": 50.2
+}
+```
+
+!!! info
+    FastAPI usa modelos Pydantic `.dict()` com <a href="https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict" class="external-link" target="_blank">seu parâmetro `exclude_unset`</a> para conseguir isso.
+
+!!! info
+    Você também pode usar:
+
+    * `response_model_exclude_defaults=True`
+    * `response_model_exclude_none=True`
+
+    conforme descrito na <a href="https://pydantic-docs.helpmanual.io/usage/exporting_models/#modeldict" class="external-link" target="_blank">documentação do Pydantic</a> para `exclude_defaults ` e `exclude_none`.
+
+#### Dados com valores para campos com valores padrão
+
+Mas se seus dados tiverem valores para os campos do modelo com valores padrão, como o item com ID `bar`:
+
+```Python hl_lines="3  5"
+{
+    "name": "Bar",
+    "description": "The bartenders",
+    "price": 62,
+    "tax": 20.2
+}
+```
+
+eles serão incluídos na resposta.
+
+#### Dados com os mesmos valores que os valores padrão
+
+Se os dados tiverem os mesmos valores que os valores padrão, como o item com ID `baz`:
+
+```Python hl_lines="3  5-6"
+{
+    "name": "Baz",
+    "description": None,
+    "price": 50.2,
+    "tax": 10.5,
+    "tags": []
+}
+```
+
+FastAPI é inteligente o suficiente (na verdade, Pydantic é inteligente o suficiente) para perceber que, embora `description`, `tax` e `tags` tenham os mesmos valores que os valores padrão, eles foram definidos explicitamente (em vez de retirados dos valores padrão) .
+
+Portanto, eles serão incluídos na resposta JSON.
+
+!!! tip
+    Observe que os valores padrão podem ser qualquer coisa, não apenas `None`.
+
+    Eles podem ser uma lista (`[]`), um `float` de `10.5`, etc.
+
+### `response_model_include` and `response_model_exclude`
+
+Você também pode usar os parâmetros do *decorador de operação de caminho* `response_model_include` e `response_model_exclude`.
+
+Eles pegam um `conjunto` de `str` com o nome dos atributos a incluir (omitindo o restante) ou a excluir (incluindo o restante).
```

Comment:
I'm not sure if this has been discussed somewhere, but I wouldn't translate `set` in this case, as it's a type - just as `list` wasn't translated in line 77:

```suggestion
Eles pegam um `set` de `str` com o nome dos atributos a incluir (omitindo o restante) ou a excluir (incluindo o restante).
```

I'd suggest doing it when it's not formatted as code. 😉